### PR TITLE
TASK-51872: Cannot remove start date of a task

### PR DESCRIPTION
**Pull request by hthabti**
issue: start date can't be removed when clicking on 'none' label from the datepicker
fix: add else-if statement to handle user's input on the datepicker when the value of entered label = 'none'

### CODE REVIEW

File: task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue [hunk 1]
```diff
@@ -492,9 +492,14 @@ export default {
             type: 'success',
             message: this.$t('alert.success.task.startDate')
           });
+          this.$root.$emit('task-start-date-updated', this.task);
         }).catch(e => {
           console.error(e);
```
Comment (rdenarie) [inline task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue]: Remove console.log

File: task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue [hunk 2]
```diff
@@ -492,9 +492,14 @@ export default {
...
-        }).finally(() => this.$root.$emit('task-start-date-updated', this.task));
+          this.$root.$emit('show-alert', {
+            type: 'error',
+            message: 'Error message to find'
```
Comment (rdenarie) [inline task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue]: Here you have to change the error message to use the good one, with i18n

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — COMMENTED** [review]
Please rebase and fixe the conflict

File: task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue [hunk 3]
```diff
@@ -492,9 +492,13 @@ export default {
             type: 'success',
             message: this.$t('alert.success.task.startDate')
           });
+          this.$root.$emit('task-start-date-updated', this.task);
         }).catch(e => {
```
Comment (hbenali) [inline task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue]:
```suggestion
        }).catch(() => {
```
`e` variable is no more used.

**Review by hbenali — APPROVED** [review]

**Pull request merged**